src/components/SearchForm.js

```diff
@@ -0,0 +1,28 @@
+import React, { Component } from 'react';
+
+class SearchForm extends Component {
+
+    handleChange = e => {
+
+    }
+
+    handleSubmit = e => {
+        e.preventDefault();
+    }
+
+
+    render() {
+        return (
+            <form onSubmit={this.handleSubmit}>
+                <input
+                    type='text'
+                    id='searchWord'
+                    value={searchWord}
+                    onChange={this.handleChange}
+                />
+                <button>Search</button>
+            </form>
+            
+        )
+    }
+}
```

Comment:
we should ask in the standup if we should continue with just a react form or if we need to incorporate redux